unfoldingWord/translationCore
# Refactor Reducers

#### This pull request addresses:
This PR removes, and refactors reducer files


#### How to test this pull request:
Make sure everything works as expected


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/1801)
